pypit/ararc.py

```diff
@@ -76,8 +77,23 @@ def detect_lines(slf, det, msarc, censpec=None, MK_SATMASK=False):
     if MK_SATMASK:
         ordwid = 0.5*np.abs(slf._lordloc[det-1] - slf._rordloc[det-1])
         msgs.info("Generating a mask of arc line saturation streaks")
-        satmask = arcyarc.saturation_mask(msarc, slf._nonlinear[det-1])
-        satsnd = arcyarc.order_saturation(satmask, ordcen, (ordwid+0.5).astype(np.int))
+#        print('calling saturation_mask')
```

Comment:
In the description of the PR, you mention that you'll remove cython code completely before merging and remove the prefix 'new_' for all the python functions. I think it's also a good idea to remove the commented code that benchmarks the routines, which I'm sure you had planned to do!

Reply:
For this PR, I was planning to commit it as is.  I wanted to remove all the cython code in one fell swoop, meaning that I want to wait until I can tests the remaining few functions.  I thought it would also be useful to have some period of time where people can use the new python code and see if there are any problems that weren't encountered by the tests in the dev suite.  So it wasn't my intention to remove the cython code, rename the python functions, and clean the code of the commented test code in this PR, but I was leaving this for one down the road.  Does that make sense, or should I go ahead and do all of this at least for the cython code that already have replacements?